front/src/pages/login/index.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import { loginUserAction } from '../../common/redux/auth/actions/auth.action';
+import { LoginButton, StyledCol, ForgotPassword, ErrorText } from './style';
+import { Form, Input, Col, Typography, Row } from 'antd';
+import 'antd/dist/antd.css';
+import {
+  validateEmail,
+  validatePassword
+} from '../../common/validators/form.validator';
+
+const { Title } = Typography;
+const layout = {
+  labelCol: { span: 8 },
+  wrapperCol: { span: 16 }
+};
+
+class LoginPage extends Component {
+  onHandleLogin = (data) => {
+    this.props.dispatch(loginUserAction(data));
```

Comment:
Please, don't use `dispatch` directly, instead, define `mapDispatchToProps` as a second argument to `connect` function

Reply:
done